project-base/app/src/DataFixtures/Demo/ComplaintDataFixture.php

```diff
@@ -43,6 +47,7 @@ public function load(ObjectManager $manager): void
             $customerUser1,
             $order1,
             $this->getReference(ComplaintStatusDataFixture::COMPLAINT_STATUS_NEW, ComplaintStatus::class),
+            $this->complaintResolutionEnum::FIX,
```

Comment:
do we use it like this? :eyes: Why not `ComplaintResolutionEnum::FIX`? I do not think the enum values would be overridden on projects :shrug: